ballista/rust/core/src/execution_plans/unresolved_shuffle.rs

```diff
@@ -69,7 +78,8 @@ impl ExecutionPlan for UnresolvedShuffleExec {
     }
 
     fn output_partitioning(&self) -> Partitioning {
-        Partitioning::UnknownPartitioning(self.partition_count)
+        //TODO the output partition is known and should be populated here!
```

Comment:
is this something that you want to finish up in this PR?

Reply:
I've filed https://github.com/apache/arrow-datafusion/issues/758 as a follow-up for implementing this since it involves more serde work.